searx/settings.yml

```diff
@@ -339,6 +339,17 @@ engines:
     disabled : True
     shortcut : or
 
+  - name : pdbe
+    engine : pdbe
+    shortcut : pdb
+# Hide obsolete PDB entries.
```

Comment:
Which content exactly? I know, if you access the site directly, and are doing a search there, the images are loaded unencrypted, but the plugin does not crawl the site. Instead it uses the pdbe api using https. The preview image is loaded over https as well.